examples/electrum_backend.rs

```diff
@@ -0,0 +1,88 @@
+use std::str::FromStr;
+
+use bdk::bitcoin::Network;
+use bdk::bitcoin::util::bip32::ExtendedPrivKey;
+use bdk::blockchain::{ElectrumBlockchain, Blockchain};
+use bdk::database::MemoryDatabase;
+use bdk::template::Bip84;
+use bdk::wallet::export::{FullyNodedExport};
+use bdk::{KeychainKind, Wallet, SyncOptions};
+
+use bdk::electrum_client::Client;
+use bdk::wallet::AddressIndex;
+use bitcoin::util::bip32;
+
+pub mod utils;
+
+use crate::utils::tx::build_signed_tx;
+
+/// This will create a wallet from an xpriv and get the balance by connecting to an Electrum server.
+/// If enough amount is available, this will send a transaction to an address.
+/// Otherwise, this will display a wallet address to receive funds.
+///
+/// This can be run with `cargo run --example electrum_backend` in the root folder.
+fn main() {
+    let network = Network::Testnet;
+
+    let xpriv = "tprv8ZgxMBicQKsPcx5nBGsR63Pe8KnRUqmbJNENAfGftF3yuXoMMoVJJcYeUw5eVkm9WBPjWYt6HMWYJNesB5HaNVBaFc1M6dRjWSYnmewUMYy";
+        "health lyrics appear aunt either wrist maple hover family episode seven maze";
+
+    let electrum_url = "ssl://electrum.blockstream.info:60002";
+
+    run(&network, electrum_url, xpriv);
+}
+
+pub fn load_or_create_wallet(
```

Comment:
This doesn't seem to be loading any wallet, and creating a new one everytime you run the example with memory database. So maybe just call it `create_wallet`??

Reply:
Done in https://github.com/bitcoindevkit/bdk/pull/526/commits/8140d130edc065362ba636828c7a8ee5afb8f884 and https://github.com/bitcoindevkit/bdk/pull/526/commits/400cbad4388322f4f0e0007365bf1a68ea95a6b0.